src/constants/config.ts

```diff
@@ -7,43 +7,43 @@ export const GRAPH_API_URL_DEVELOP_GOERLI =
   'https://api.thegraph.com/subgraphs/name/gnosis-auction/ga-goerli'
```

Comment:
Change "gnosis" here to "fp"?

Reply:
Since we've branched out of Gnosis Auction code, I've kept the dev apis as the same ga ones and fp ones as production apis